models/articles-model.js

```diff
@@ -16,4 +16,14 @@ exports.selectArticles = () => {
     return db.query('SELECT * FROM articles;').then((result) => {
       return result.rows;
     });
-  };
\ No newline at end of file
+};
+
+exports.updateArticleVotes = ({article_id, votes}) => {
+    if (!Number.isInteger(votes)) {
+        return Promise.reject({ status: 400, msg: 'Invalid votes value. Must be an integer.' });
+    }
```

Comment:
It's not necessary to check this manually - see what happens if you remove this. Because it violates the seed psql should error itself when you attempt the query and end up in your catch block!